src/lib/strings/url-helpers.ts

```diff
@@ -167,6 +168,9 @@ export function convertBskyAppUrlIfNeeded(url: string): string {
     } catch (e) {
       console.error('Unexpected error in convertBskyAppUrlIfNeeded()', e)
     }
+  } else if (!isWeb && isShortLink(url)) {
+    // We only want to do this on native, web handles the 301 for us
+    return shortLinkToHref(url)
```

Comment:
We don't need to do this for web, so only do it on native. Web can handle the 301 elegantly without weirdness.